extensions/json-language-features/package.nls.json

```diff
@@ -6,7 +6,7 @@
 	"json.schemas.fileMatch.desc": "An array of file patterns to match against when resolving JSON files to schemas. `*` can be used as a wildcard. Exclusion patterns can also be defined and start with '!'. A file matches when there is at least one matching pattern and the last matching pattern is not an exclusion pattern.",
 	"json.schemas.fileMatch.item.desc": "A file pattern that can contain '*' to match against when resolving JSON files to schemas.",
 	"json.schemas.schema.desc": "The schema definition for the given URL. The schema only needs to be provided to avoid accesses to the schema URL.",
-	"json.format.enable.desc": "Enable/disable default JSON formatter",
+	"json.format.enable.desc": "Enables the default JSON formatter",
 	"json.tracing.desc": "Traces the communication between VS Code and the JSON language server.",
 	"json.colorDecorators.enable.desc": "Enables or disables color decorators",
```

Comment:
This one's sneaky

Reply:
Wouldn't this still read the same?

- [ ] Enables the default JSON formatter == off
- [x] Enables the default JSON formatter == on

Reply:
I meant line 11: `"json.colorDecorators.enable.desc": "Enables or disables...`